### PR TITLE
Add s1ck/graph library

### PR DESCRIPTION
**Pull request by s1ck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,6 +259,10 @@ See also [Games Made With Piston](https://github.com/PistonDevelopers/piston/wik
 * [Veloren](https://gitlab.com/veloren/veloren) — An open world, open source multiplayer voxel RPG game currently in alpha development [![build badge](https://gitlab.com/veloren/veloren/badges/master/pipeline.svg)](https://gitlab.com/veloren/veloren/-/pipelines)
 * [Zone of Control](https://github.com/ozkriff/zoc) — A turn-based hexagonal strategy game [![build badge](https://api.travis-ci.org/ozkriff/zoc.svg?branch=master)](https://travis-ci.org/ozkriff/zoc)
 
+### Graph theory
```
Comment (Israel-Laguan) [inline README.md:262]: Not sure if this can be placed in Macro or somewhere else, but no issues anyway

**Review by Israel-Laguan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (s1ck) [reply README.md:262]: Mh, I don't think it fits in with the crates listed under `Macro`. We can rename the category to `Graph algorithms`. I didn't want to use just `Graphs` as it's often mistaken as graph diagrams.

Comment (Israel-Laguan) [reply README.md:262]: Maybe "algorithms" sounds better

Comment (s1ck) [reply README.md:262]: As in `Graph algorithms` ?

**Comment by s1ck** [comment]
@Israel-Laguan do you have an idea why the check is failing?

**Comment by Israel-Laguan** [comment]
> @Israel-Laguan do you have an idea why the check is failing?

As I understand an admin should trigger the actions, like @palfrey

**Comment by palfrey** [comment]
> > @Israel-Laguan do you have an idea why the check is failing?
> 
> https://github.com/rust-unofficial/awesome-rust/actions/runs/3489674571/jobs/5847365420#step:6:1
> 
> So seems like the linter identifies this difference in the TOC as weird, maybe an admin should check, like @palfrey

It's complaining because the new section is in the Applications area, but the ToC bit got added under Libraries

**Pull request merged**